neard/src/config.rs

```diff
@@ -13,6 +13,8 @@ use log::info;
 use num_rational::Rational;
 use serde::{Deserialize, Serialize};
 
+#[cfg(feature = "delay_detector")]
```

Comment:
Wait, we removed `, optional = true` from Cargo.tom, so ther's no longer `delay_detector` feature I think

Reply:
We removed `, optional = true` from `delay_detector` package, the `delay_detector` feature is independent from the package with the same name.

We were not using `delay_detector` package inside neard before, I was getting compilation errors with `, optional = true`. Anyway, I moved this code to another PR.